sprint-planning/readme.md

```diff
@@ -1,29 +1,83 @@
 # Planning
 
-## Content TODO's
+## Goals
 
-- Complete outline below
-- Determining team capacity
-- Tasking sprint stories
-  - size of tasks
-- WIP limits / dividing up into workstreams
+During the [sprint planning](https://www.agilealliance.org/glossary/sprint-planning), the team discusses and agrees on the scope for the upcoming sprint.
 
-## Goals
+**Goals are:**
+
+- Selecting the **stories** that will be implemented in the sprint.
+- Estimating the **effort** required for the stories in the sprint.
+- Splitting the stories into **tasks**.
+
+General guidance:
 
-What are the intended outcomes of the ceremonies?
+- The sprint planning should happen at the begining of the new sprint (or at the end of the previous one). It usually last bewteen 1 and 4 hours depending on the size of the team and duration of the sprints.
+- Each story should be achievable in the sprint. Otherwise, you should split the story.

```

Comment:
Nit: "Each story should be able to be completed within the length of the sprint. Otherwise, the story should be broken up into multiple stories."

Reply:
Fixed in upcoming PR update